### ISSUE TITLE
// reverses the order of selection

### ISSUE THREAD

**Issue by mitchellwrosen** [fimad/scalpel#54]
Hi, I discovered that calling something like...

```
htmls x
```

will match results in the opposite order as

```
htmls (x // y)
```

(assuming that there is one `y` inside each `x`).

Is this intended? Thanks!

**Comment by fimad** [comment]
That is not intentional, I would have expected the the same ordering.

**Comment by fimad** [comment]
I cannot seem to reproduce this:

```haskell
ghci> let html = "<a><b>1</b></a><a><b>2</b></a><a><b>3</b></a>"
ghci> scrapeStringLike html $ texts "a"
Just ["1","2","3"]
ghci> scrapeStringLike html $ texts ("a" // "b")
Just ["1","2","3"]
```

Which version of scalpel are you using? Do you have a specific example that exhibits the problem?

**Comment by tetotechy** [comment]
the problem arises when you use the `"b"` selector:
```haskell
ghci> let html = "<a><b>1</b></a><a><b>2</b></a><a><b>3</b></a>"
ghci> scrapeStringLike html $ texts (tagSelector "b")
Just ["3","2","1"]
```

**Comment by fimad** [comment]
It looks like this may be fixed with #59.

I just tried patching the proposed fix in #59 and it results in `Just ["1","2","3"]` for all 3 cases.

**Comment by fimad** [comment]
This should now be fixed in v0.5.1.
